packages/harpstrata/src/get-harp-strata/utils/get-degree-matrix/get-degree-matrix.test.ts

```diff
@@ -24,3 +25,10 @@ test('getDegreeMatrix maps a major diatonic halfstepmatrix in to a major diatoni
 
   expect(actualArray).toStrictEqual(MAJOR_DIATONIC_FIRST_POZITION)
 })
+
+test('getDegreeMatrix function maps a simple 2d array of -1s to 7th degrees (11) when halfsetp offset is 0', () => {
+  const expectedArray = [[seventh], [seventh]]
+  const actualArray = getDegreeMatrix([[-1], [-1]], 0)
+
+  expect(actualArray).toStrictEqual(expectedArray)
+})
```

Comment:
Can add a few more negative tests here just to make sure that other positions work etc.

Reply:
Done